tstest/integration/vms/vms_test.go

```diff
@@ -49,6 +49,7 @@ var distroRex *regexValue = func() *regexValue {
 	flag.Var(result, "distro-regex", "The regex that matches what distros should be run")
 	return result
 }()
+var downThemAll = flag.Bool("download-all-images", false, "if set, download every distro image we know about")
```

Comment:
doesn't it download it anyway as a part of testing them?

Reply:
Yes, but the download takes a while and can go over the default test timeout.

Reply:
This basically warms the cache.

Reply:
But it's misleading, because even if it's false, it still downloads them and warms the cache.... it just does so why failing to run the test also.

How about you remove the flag, add a new TestDownloadImages that just downloads, and then it can be run as:

```
     go test --run=DownloadImages
```

Also, it should also support the distro-regex flag.

Reply:
https://github.com/tailscale/tailscale/commit/4fe5a18fad65d1585041c42b82a928f81919f30d has this behavior